app/models/service.rb

```diff
@@ -8,6 +8,20 @@ class Service < ApplicationRecord
 
   validates :hours, numericality: { greater_than: 0 }, presence: true
 
+  rails_admin do
+    list do
```

Comment:
Order here should probably be event, student, hours, I think

Reply:
Updated.